pyquil/operator_estimation.py

```diff
@@ -706,13 +710,20 @@ def group_experiments(experiments: TomographyExperiment,
 @dataclass(frozen=True)
 class ExperimentResult:
     """An expectation and standard deviation for the measurement of one experiment setting
-    in a tomographic experiment.
+    in a tomographic experiment. In the case of readout error calibration, we also include
```

Comment:
typically the first line of a docstring should be 1(ish) line and further details should be in subsequent paragraphs

Reply:
Updated.